x-pack/plugins/security_solution/server/lib/timeline/routes/README.md

```diff
@@ -323,4 +323,292 @@ kbn-version: 8.0.0
      "timelineId":"f5a4bd10-83cd-11ea-bf78-0547a65f1281", // This is a must as well
      "version":"Wzg2LDFd" // Please provide the existing timeline version
 }
-```
\ No newline at end of file
+```
+
+## Export timeline api
+
+#### POST /api/timeline/_export
+
+##### Authorization
+
+Type: Basic Auth
+
+username: Your Kibana username
+
+password: Your Kibana password
+
+
+
+
+##### Request header
+
+```
+
+Content-Type: application/json
+
+kbn-version: 8.0.0
+
+```
+
+##### Request param
+file_name	${filename}.ndjson
+
+
+##### Request body
+```json
+{
+	ids: [
+		${timelineId}
+	]
+}
+```
+
+## Import timeline api
+
+#### POST /api/timeline/_import
+
+##### Authorization
+
+Type: Basic Auth
+
+username: Your Kibana username
+
+password: Your Kibana password
+
+
+
+
+##### Request header
+
+```
+
+Content-Type: application/json
+
+kbn-version: 8.0.0
+
+```
+
+##### Request body
+a .ndjson file
```

Comment:
I think you should put here:
```
file: sample.ndjson
```
since you need the file key. i just tried sending raw json